metadata/src/main/java/org/apache/kafka/metadata/placement/StripedReplicaPlacer.java

```diff
@@ -33,89 +33,86 @@
 
 /**
  * The striped replica placer.
- *
- *
- * GOALS
- * The design of this placer attempts to satisfy a few competing goals.  Firstly, we want
- * to spread the replicas as evenly as we can across racks.  In the simple case where
- * broker racks have not been configured, this goal is a no-op, of course.  But it is the
+ * <p>
+ * <h3>Goals</h3>
+ * <p>The design of this placer attempts to satisfy a few competing goals. Firstly, we want
+ * to spread the replicas as evenly as we can across racks. In the simple case where
+ * broker racks have not been configured, this goal is a no-op, of course. But it is the
  * highest priority goal in multi-rack clusters.
```

Comment:
I think the JavaDoc parser is permissible and won't complain about it, but here we are missing a `</p>`